tests/test_json_model.py

```diff
@@ -143,8 +144,21 @@ def test_user_agent(self):
         headers, path_params, query_params, body)
 
     self.assertEqual(headers['user-agent'],
-        'my-test-app/1.23.4 google-api-python-client/' + __version__ +
-        ' (gzip)')
+        'my-test-app/1.23.4')
+
+  def test_x_goog_api_client(self):
+    model = JsonModel(data_wrapper=False)
+
+    headers = {'x-goog-api-client': 'gccl/1.23.4'}
+    path_params = {}
+    query_params = {}
+    body = {}
+
+    headers, unused_params, unused_query, body = model.request(
+        headers, path_params, query_params, body)
+
+    self.assertEqual(headers['x-goog-api-client'],
+        'gccl/1.23.4' + ' gdcl/' + __version__ + ' gl-python/' + platform.python_version())
```

Comment:
re: `gccl/...` Will this make it impossible to disambiguate Google Cloud client libraries from Google API client libraries?

Reply:
`gdcl` is for discovery clients, and `gccl` is for the cloud client libraries. We'd like it to be possible for cloud clients that wrap this library to pass extra headers. See line 152

Reply:
Looks like there was no change to add gccl.  Presumably that is existing behavior?  And may not be correct since this is not a Veneer client library?

Reply:
oh, I see - it's part of the test.  nevermind :)

Reply:
maybe add a comment indicating that this is testing composition of clients?